DANS-KNAW/easy-deposit-api
# EASY-2537: upload files to data/original/*

Fixes EASY-2537 upload to original

#### When applied it will
* [x] merge `StagedFilesTarget` into `DataFiles`
* [x] make compatible with existing datasets without the origin folder
* 

#### Where should the reviewer @DANS-KNAW/easy start?

#### How should this be manually tested?

* [x] create two bags and upload attached zip
* delete files outside the folder "original" from the second bag
* build and deploy
* upload another file to the both bags
* create another bag, upload something
* submit the three bags
* check the result in the files tab of the archived datasets

after deployment, the folder "original" will still show up in the first draft but no longer in the second.


![image](https://user-images.githubusercontent.com/10553630/74437600-ac26a300-4e68-11ea-8920-703c0d596932.png)


#### Related pull requests on github
repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#](PRlink)     | ..
